crates/biome_css_analyze/src/lint/nursery/no_missing_var_function.rs

```diff
@@ -0,0 +1,247 @@
+use biome_analyze::{context::RuleContext, declare_lint_rule, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_css_syntax::{AnyCssProperty, CssDashedIdentifier, CssDeclaration, CssSyntaxKind};
+use biome_rowan::AstNode;
+
+use crate::services::semantic::Semantic;
+
+declare_lint_rule! {
+    /// Disallow missing var function for css variables.
+    ///
+    /// This rule has the following limitations:
+    /// - It only reports custom properties that are defined within the same source.
+    /// - It does not check properties that can contain author-defined identifiers. It ignores the following
+    ///   - `animation`
+    ///   - `animation-name`
+    ///   - `counter-increment`
+    ///   - `counter-reset`
+    ///   - `counter-set`
+    ///   - `grid-column`
+    ///   - `grid-column-end`
+    ///   - `grid-column-start`
+    ///   - `grid-row`
+    ///   - `grid-row-end`
+    ///   - `grid-row-start`
+    ///   - `list-style`
+    ///   - `list-style-type`
+    ///   - `transition`
+    ///   - `transition-property`
+    ///   - `view-transition-name`
+    ///   - `will-change`
```

Comment:
```suggestion
    /// - It does not check properties that can contain author-defined identifiers. 
    /// - Ignored properties include:
    ///   - `animation`
    ///   - `animation-name`
    ///   - `counter-increment`
    ///   - `counter-reset`
    ///   - `counter-set`
    ///   - `grid-column`
    ///   - `grid-column-end`
    ///   - `grid-column-start`
    ///   - `grid-row`
    ///   - `grid-row-end`
    ///   - `grid-row-start`
    ///   - `list-style`
    ///   - `list-style-type`
    ///   - `transition`
    ///   - `transition-property`
    ///   - `view-transition-name`
    ///   - `will-change`
```

You may can add line break?